packages/template-retail-react-app/app/components/social-login/index.jsx

```diff
@@ -55,9 +55,9 @@ const SocialLogin = ({idps}) => {
                                 onClick={() => {
                                     alert(message)
                                 }}
-                                borderColor="gray.500"
```

Comment:
removing this causes the social login buttons in the /login page to be blue. Could we make this passed in as a prop?
<img width="481" alt="Screenshot 2024-10-31 at 3 31 21 PM" src="https://github.com/user-attachments/assets/31936866-0032-4407-bc17-8c1cac2bfef4">